### ISSUE TITLE
Can't restart a download after cancelling

### ISSUE THREAD

**Issue by scottsab** [rmcrackan/Libation#1001]
After cancelling a download in progress due to a Libation crash message, there was no way to restart it. The traffic light remained yellow no matter what I did. I would like to restart the download of this book. 
teamdba@yahoo.com

**Comment by rmcrackan** [comment]
Sorry about this. Does it help if you right click and set status to 'not downloaded' and then retry?

(I'll also email this)

**Comment by scottsab** [comment]
When I right clicked, the 'not downloaded' option was grayed out. However, when I exited the application and then later started it up again, I was able to resume the download on that book. Thanks for your reply!
--Scott


    On Monday, October 14, 2024 at 08:40:34 AM PDT, rmcrackan ***@***.***> wrote:  
 
 


Sorry about this. Does it help if you right click and set status to 'not downloaded' and then retry?

(I'll also email this)
